src/index.html

```diff
@@ -207,7 +207,7 @@ <h3>New furniture</h3>
         </div>
       </div>
       <div class="row">
-        <div class="col-3">
+        <div class="col-12 col-sm-6 col-xl-4">
```

Comment:
Ja akurat uważam, że wygląda to dobrze, ale zmniejszyłeś liczbę elementów w wierszu w najwyższych rozdzielczościach z 4 do 3, przez co nie jest to zgodne z projektem (plik .psd w treści modułu 21.4)